lib/src/server/server.dart

```diff
@@ -163,15 +163,6 @@ class Server extends ConnectionServer {
   ServerSocket? _insecureServer;
   SecureServerSocket? _secureServer;
 
-  /// Create a server for the given [services].
```

Comment:
This probably requires bump to `4.0.0` and the same below.

Reply:
Right..